flink-python/bin/pyflink-udf-runner.sh

```diff
@@ -32,16 +32,35 @@ if [[ "$python" = "" ]]; then
     python="python"
 fi
 
-# Add pyflink & py4j to PYTHONPATH
-PYFLINK_ZIP="$FLINK_OPT_DIR/python/pyflink.zip"
-if [[ ! ${PYTHONPATH} =~ ${PYFLINK_ZIP} ]]; then
+CURRENT_DIR=`pwd -P`
```

Comment:
This script is running on worker. Add some comments that why we add the code here. I spend some time to figure out that it is added for debug friendly.